src/kili/entrypoints/mutations/project/import_csv.py

```diff
@@ -0,0 +1,141 @@
+"""Import from csv module."""
+import csv
+import logging
+from typing import Dict, List, Optional, Tuple
+
+from kili.services.types import LabelType
+
+logging.basicConfig()
+logger = logging.getLogger("kili.entrypoints.mutations.project.import_csv")
+logger.setLevel(logging.INFO)
+
+
+# pylint: disable=too-many-arguments,too-many-locals
+def import_text_assets_labels_from_csv(
+    kili,
+    csv_file: str,
+    content_column: str,
+    unique_categories: Optional[List[str]],
+    category_column: Optional[str],
+    external_id_column: Optional[str],
+    csv_separator: str,
+    label_type: LabelType,
+    project_id: Optional[str],
+    project_title: Optional[str],
+    project_description: Optional[str],
+) -> str:
+    """Import a csv file into a text classification project."""
+    logger.info(f"Reading csv file: {csv_file}")
+    content_array, categories_array, external_id_array = _read_csv(
+        csv_file, content_column, category_column, external_id_column, csv_separator
+    )
+
+    if project_id is None:
+        if project_title is None:
+            raise ValueError("`project_title` is required if `project_id` is not provided.")
+
+        if project_description is None:
+            raise ValueError("`project_description` is required if `project_id` is not provided.")
+
+        if unique_categories:
+            categories = unique_categories
+        elif categories_array:
+            categories = list(set(categories_array))
```

Comment:
we could raise an error also if categories_array is not included in unique_categories